bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/editor/targetComponent/details/ValidAtEditor.scala

```diff
@@ -109,10 +109,9 @@ abstract class ValidAtEditor[A <: ITarget](empty: A) extends TelescopePosEditor
     node = node0
     nonreentrant {
       val nst = spt.getTarget.asInstanceOf[NonSiderealTarget]
-      Option(nst.getDateForPosition).foreach { d =>
-        calendar.setDate(d)
-        timeConfig.setTime(d)
-      }
+      val d   = Option(nst.getDateForPosition) | new java.util.Date
```

Comment:
I made this change because otherwise a newly created non-sidereal target would display the date for the last non-sidereal target that happened to be selected.